packages/duckdb-wasm/src/bindings/runtime_browser.ts

```diff
@@ -395,9 +395,11 @@ export const BROWSER_RUNTIME: DuckDBRuntime & {
                             mod.HEAPU8.set(src, buf);
                             return src.byteLength;
                         } else if (xhr.status == 200) {
-                            throw new Error(
-                                `Range request for ${file.dataUrl} did not return a partial response: ${xhr.status} "${xhr.statusText}"`,
-                            );
+                            // TODO: here we are actually trowing away all non-relevant bytes, but this is still better than failing
```

Comment:
nit: typo in `trowing`

as discussed on discord already, we probably want a warning for the user here